db/migrate/20200417153503_add_unconfirmed_email_to_spree_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddUnconfirmedEmailToSpreeUsers < ActiveRecord::Migration[6.0]
```

Comment:
We should use our helper for migrations here. I think specs are failing on old Rails versions for this reason.

Reply:
@kennyadsl correct me if I'm wrong, but we shouldn't need the helper anymore. `ActiveRecord::Migration[5.2]` will work with Rails 5.2+, and we don't support anything older than 5.2. Fixing now!